proxy/plugin/filter_by_url_regex.py

```diff
@@ -92,10 +89,6 @@ def handle_client_request(
                     headers={b'Connection': b'close'},
                     reason=b'Blocked',
                 )
-                # stop looping through filter list
-                break
```

Comment:
This removes unreachable code. Could you also try dropping the linter exclusions at https://github.com/abhinavsingh/proxy.py/blob/7c5877b/.pylintrc#L129 and https://github.com/abhinavsingh/proxy.py/blob/7c5877b/.flake8#L149 to see if this was the only place with this type of a linting violation?